### PR TITLE
ESLint #1 - timesignature, dot, textnote, tabnote

### PR DESCRIPTION
**Pull request by Silverwolf90**

### CODE REVIEW

**Comment by Silverwolf90** [comment]
Ok these PR's should all be good now. I tried to set it up so that the build incrementally passes with each chunk. And you should be able to merge each in succession. Hopefully this goes as well as I hope :P

**Comment by 0xfe** [comment]
Thanks. There are two "#4" PRs -- which one first?

On Sun, Jul 10, 2016 at 8:03 AM, Cyril Silverman <notifications@github.com>
wrote:

> Ok these PR's should all be good now. I tried to set it up so that the
> build incrementally passes with each chunk. And you should be able to merge
> each in succession. Hopefully this goes as well as I hope :P
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/0xfe/vexflow/pull/390#issuecomment-231585539>, or mute
> the thread
> <https://github.com/notifications/unsubscribe/AAOuk2rDwHzlybhZy192iWlEesJIDTeaks5qUN-kgaJpZM4JI0bb>
> .
>



-- 
Mohit Muthanna [mohit (at) muthanna (uhuh) com]

**Comment by Silverwolf90** [comment]
Fixed numbering.

**Comment by 0xfe** [comment]
Looks like there was a formatting regression here (maybe width calculation, or padding/spacing) in TextNote:

textnote textnote_formatting_with_glyphs_1_blessed
textnote textnote_formatting_with_glyphs_1_current
textnote textnote_formatting_with_glyphs_1
textnote textnote_formatting_blessed
textnote textnote_formatting_current
textnote textnote_formatting
textnote textnote_formatting_with_glyphs_0_blessed
textnote textnote_formatting_with_glyphs_0_current
textnote textnote_formatting_with_glyphs_0

File: src/textnote.js [hunk 1]
```diff
@@ -208,23 +209,25 @@ export class TextNote extends Note {
...
+      throw new Vex.RERR('NoRenderContext', "Can't measure text without rendering context.");
+    }
+
     if (this.preFormatted) return;
 
     if (this.smooth) {
       this.setWidth(0);
     } else {
-      if (this.glyph) {
+      if (!this.glyph) {
```
Comment (0xfe) [inline src/textnote.js]: Is this logical negation intentional?

Comment (0xfe) [inline src/textnote.js]: Okay, yes, undoing this fixed the regression tests.

**Comment by Silverwolf90** [comment]
Hmm, I wonder why these didn't show up in Travis?

**Comment by 0xfe** [comment]
Yeah, that's weird, not sure. Will dig in after these changes are commited.

On Sun, Jul 10, 2016 at 9:12 AM, Cyril Silverman <notifications@github.com>
wrote:

> Hmm, I wonder why these didn't show up in Travis?
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/0xfe/vexflow/pull/390#issuecomment-231588365>, or mute
> the thread
> <https://github.com/notifications/unsubscribe/AAOuk-1Pfc-YkE_vWFqDD1CMJEIwZTIvks5qUO-ygaJpZM4JI0bb>
> .
>



-- 
Mohit Muthanna [mohit (at) muthanna (uhuh) com]

**Pull request merged**